docs/modules/ROOT/pages/how-tos/upgrade-5.x-to-6.x.adoc

```diff
@@ -0,0 +1,18 @@
+= Upgrade from v5 to v6
+
+This guide describes the steps to perform an upgrade of the component from version v5 to v6.
+
+== Changes
+
+Configuration of TLS aspect has been overhauled:
+* Separation of internal (ingress to Keycloak) and external (client to ingress) TLS configuration
+
+
+
+== Step-by-step guide
+
+When upgrading the component, the following changes to the inventory are required:app-name:
```

Comment:
```suggestion
When upgrading the component, the following changes to the inventory are required:
```

Reply:
My editor trying to be a smart-ass, gets me every time 😂